smarts/core/motion_planner_provider.py

```diff
@@ -29,6 +29,8 @@
 
 
 class MotionPlannerProvider(Provider):
+    _poses: np.ndarray
```

Comment:
What was the intent here?  (`MotionPlannerProvider._poses` is not the same thing as `self._poses` below.  Is this used anywhere?)

Reply:
pytype throws attribute error on `self._poses`. See [here](https://google.github.io/pytype/errors.html#attribute-error) for explanation on attribute error and suggested solution.

Reply:
Ah ok, I guess that makes sense.  Thanks for the link.  

Maybe a short comment here might prevent someone else thinking this is extraneous and removing it in the future.

(Personally, I'm not a big fan of cluttering the code just to for `pytype`, but I guess this is just a matter of opinion.)